addons/web/views/webclient_templates.xml

```diff
@@ -409,11 +409,15 @@
                     <t t-esc="message"/>
                 </p>
 
-                <input type="hidden" name="redirect" t-att-value="redirect"/>
                 <div t-attf-class="clearfix oe_login_buttons text-center mb-1 {{'pt-2' if form_small else 'pt-3'}}">
                     <button type="submit" class="btn btn-primary btn-block">Log in</button>
+                    <t t-if="debug">
```

Comment:
why do you need this in login form?

Reply:
@simahawk it's a safety measure. Just in case the main view is broken for regular users, so you can log in and jump right away to the main view as the superuser.

Reply:
Although you are right, it is not strictly needed.
You can simply open the url `/web/become` and log in.
The login redirection goes to `/web/become`, which makes you become superuser, and voilà!

Reply:
so what? you just get to `/web/login?debug=1` and you are able to login as superuser?

Reply:
and judging from the code of the controller: you must have `_is_system` true which means that you have to be logged in which means that you won't get to the login page because when you are logged in you get redirect to home... am I wrong?

Reply:
hmm I'm pretty sure it used to redirect... :thinking:  but seems not... :open_mouth: 
and it actually sounds akward that you can get here while being logged in

![image](https://user-images.githubusercontent.com/347149/46204344-37a79e80-c31d-11e8-879c-816ecc8af3fe.png)

So, all in all, the button seems good but then since you have it there I'd use a POST request to avoid random calls to the controller.

Reply:
> and judging from the code of the controller: you must have _is_system true which means that you have to be logged in which means that you won't get to the login page because when you are logged in you get redirect to home... am I wrong?
 
Yes. The button will go to the regular login endpoint *then* redirect to the "become superuser" endpoint, instead of the usual redirection to /web.